newIDE/app/src/ObjectEditor/Editors/SpriteEditor/SpritesList.js

```diff
@@ -189,8 +189,7 @@ export default class SpritesList extends Component {
         project,
         direction.getSprite(i).getImageName()
       );
-      var importedImage = new Image();

```

Comment:
@blurymind This importedImage wasn't useful right? I've removed it and everything still work.

Reply:
I must have forgoten it there after doing some tests